src/api/democracy.ts

```diff
@@ -19,6 +20,26 @@ const CONVICTIONS_BLOCKS: { [key: string]: number } = {
   locked6x: 1612800,
 }
 
+const voteConviction = (vote?: PalletDemocracyVoteAccountVote): String => {
+  if (vote?.isSplit) {
+    return 'None'
+  } else if (vote?.isStandard) {
+    return vote.asStandard?.vote.conviction.toString()
+  } else {
+    return 'None'
+  }
+}
+
+const voteAmount = (vote?: PalletDemocracyVoteAccountVote): BN => {
+  if (vote?.isSplit) {
+    return vote.asSplit.toBigNumber() + vote.asSplit.nay.toBigNumber()
+  } else if (vote?.isStandard) {
+    return vote.asStandard.balance.toBigNumber()
+  } else {
+    return BN_0
```

Comment:
I honestly don't know if it better to return undefined or a default value here in ReactJS